human-eval-benchmark/run_humaneval.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+JOB_YAML=./manifests/job.yaml
+JOB_NAME=human-eval-deployment-job
+NAMESPACE=default
```

Comment:
Can you provide an option to change the namespace as parameter. By default, the namespace can be 'default'

Reply:
@kusumachalasani added it!